pkg/agent/main.go

```diff
@@ -50,7 +54,7 @@ func main() {
 	}
 
 	if err != nil {
-		log.Fatal(err)
+		logrus.Fatal(err)
```

Comment:
Curious why we are using both log and logrus in this package?

Reply:
Fair enough, removed `log`